test/cmdlineTests/name_simplifier/output

```diff
@@ -139,14 +139,14 @@ object "C_59" {
             }
             function storage_array_index_access_struct_S_1252() -> slot, offset
             {
-                if iszero(lt(slot, 0x02)) { panic_error_0x32() }
-                slot := add(slot, slot)
+                if iszero(lt(offset, 0x02)) { panic_error_0x32() }
```

Comment:
Random comment: why aren't these checks removed? `offset` is always 0.

Reply:
@axic This is a known issue. CommonSubExpression eliminator replaces `0` by return variables. And ExpressionSimplifier is not able to simplify this. There are two PRs proposing a fix https://github.com/ethereum/solidity/pull/11246/ and ttps://github.com/ethereum/solidity/pull/11243